docs/api/afhsb.md

```diff
@@ -0,0 +1,72 @@
+# About
+
+This is the documentation of the API for accessing the AFHSB (`afhsb`) data source of
+the [Delphi](https://delphi.cmu.edu/)'s epidemiological data.
+
+## AFHSB Data
+
+... <!-- TODO -->
+
+## Contributing
+
+If you are interested in contributing:
+
+- For development of the API itself, see the
+  [development guide](docs/epidata_development.md).
+- To suggest changes, additions, or other ways to improve,
+  [open an issue](https://github.com/cmu-delphi/delphi-epidata/issues/new)
+  describing your idea.
+
+## Citing
+
+We hope that this API is useful to others outside of our group, especially for
+epidemiological and other scientific research. If you use this API and would
+like to cite it, we would gratefully recommend the following copy:
+
+> David C. Farrow,
+> Logan C. Brooks,
+> Aaron Rumack,
+> Ryan J. Tibshirani,
+> Roni Rosenfeld
+> (2015).
+> _Delphi Epidata API_.
+> https://github.com/cmu-delphi/delphi-epidata
+
+## Data licensing
+
+The data surfaced through this API is just a carefully curated mirror of data acquired from .... <!-- TODO -->
+It is subject to its original licensing, .... <!-- TODO -->
+
+# The API
+
+The base URL is: https://delphi.midas.cmu.edu/epidata/api.php
+
+See [this documentation](README.md) for details on specifying epiweeks, dates, and lists.
+
+## Parameters
+
+### Required
+
+| Parameter | Description | Type |
+| --- | --- | --- |
+| `auth` | password | string |
+| `epiweeks` | epiweeks | `list` of epiweeks |
+| `locations` | locations | `list` of [region](labels/regions.txt), [state](labels/states.txt), or 3-letter country code labels |
```

Comment:
here, similar ones in other docs: these relative links are broken (i think) now that this is in the `docs/` dir

Reply:
did a replace-all for "(labels/" -> "(../../labels/" and "(src/" -> "(../../src/".